kvdb/etcd/embed.go

```diff
@@ -68,7 +68,7 @@ func NewEmbeddedEtcdInstance(path string, clientPort, peerPort uint16,
 	cfg.Dir = path
 
 	// To ensure that we can submit large transactions.
-	cfg.MaxTxnOps = 8192
+	cfg.MaxTxnOps = 16384
```

Comment:
I suppose this is just to get the tests pass, but does this text in `etcd.md` need changes?

>The large max-txn-ops and max-request-bytes values are currently required in case of running LND with the full graph in etcd. Upcoming versions will split the database to local and replicated parts and only essential parts will remain in the replicated database, removing the requirement for these additional settings. These parameters have been tested to work with testnet LND.

Reply:
Did a small update on the `etcd.md` doc.